packages/components/src/components/inline-message/inline-message.scss

```diff
@@ -0,0 +1,79 @@
+.inline-message {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 0.8rem;
+    gap: 0.8rem;
+    border-radius: $BORDER_RADIUS;
+    height: min-content;
+    width: fit-content;
+
+    &__lg {
+        @include tablet-up {
+            padding: 1.6rem;
+            gap: 1.6rem;
+            border-radius: $BORDER_RADIUS * 2;
+        }
+    }
+
+    &__warning {
+        background: rgba(255, 173, 58, 0.16);
```

Comment:
@farzin-deriv this color is used in other places in the project, let's define it in the constants.scss and reuse it everywhere